README.md

```diff
@@ -33,45 +33,75 @@ sample rokudeploy.json
     "password": "securePassword"
 }
 ```
-## Usage
+## CLI Usage
 
-From a node script
-```javascript
-var rokuDeploy = require('roku-deploy');
+### Deploy a zip package
+Deploy a .zip package of your project to a roku device
+```shell
+npx roku-deploy deploy --host ${ip-of-roku} --password ${password} --rootDir '.' --outDir './out'
+```
 
-//deploy a .zip package of your project to a roku device
-rokuDeploy.deploy({
-    host: 'ip-of-roku',
-    password: 'password for roku dev admin portal'
-    //other options if necessary
-}).then(function(){
-    //it worked
-}, function(error) {
-    //it failed
-    console.error(error);
-});
+
+### Create a signed package of your project
+```shell
+npx roku-deploy deploy package --host ${ip-of-roku} --password ${password} --signingPassword ${signing-password}
 ```
-Or
-```javascript
-//create a signed package of your project
-rokuDeploy.deployAndSignPackage({
-    host: 'ip-of-roku',
-    password: 'password for roku dev admin portal',
-    signingPassword: 'signing password'
-    //other options if necessary
-}).then(function(pathToSignedPackage){
-    console.log('Signed package created at ', pathToSignedPackage);
-}, function(error) {
-    //it failed
-    console.error(error);
-});
+
+### Other uses
+Stage the root directory
+```shell
+npx roku-deploy stage --stagingDir ${stagingDir} --rootDir ${rootDir}
```

Comment:
The `${}` pattern is a little confusing. Perhaps use actual values, but make them obvious. Like this:

```shell
npx roku-deploy stage --stagingDir "./path/to/staging/dir" --rootDir "./path/to/root/dir"